eya_def_tools/eya_def_tools/data_models/general.py

```diff
@@ -67,16 +67,56 @@ class Organisation(EyaDefBaseModel):
     )
 
 
-class AssessmentBasis(StrEnum):
-    """Basis on which an EYA or WRA component has been assessed."""
+AssessmentComponentBasis = Literal[
+    "time_series_calculation",
+    "distribution_calculation",
+    "other_calculation",
+    "project_specific_assumption",
+    "regional_assumption",
+    "generic assumption",
+    "not_considered",
+]
 
-    TIME_SERIES_CALCULATION = auto()
-    DISTRIBUTION_CALCULATION = auto()
-    OTHER_CALCULATION = auto()
-    PROJECT_SPECIFIC_ASSUMPTION = auto()
-    REGIONAL_ASSUMPTION = auto()
-    GENERIC_ASSUMPTION = auto()
-    NOT_CONSIDERED = auto()
+
+AssessorType = Literal[
```

Comment:
We concluded during the EYA DEF group meeting yesterday that the terms "first-party," "second-party," and "third-party" may not be entirely clear. They may be misinterpreted in terms of a project transaction, where a consultant undertaking an EYA is typically a third party. It should be better to use the terms "issuing organisation," "receiving organisation" and "other organisation." Potentially there could be an option for "commissioning organisation" also for cases when that is different from the "receiving organisation."